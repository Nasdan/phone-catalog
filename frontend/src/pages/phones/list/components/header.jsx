import React from 'react';

export const Header = props => <h1>Phone Catalog</h1>;
