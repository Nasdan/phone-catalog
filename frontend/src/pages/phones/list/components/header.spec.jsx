import React from 'react';
import { shallow } from 'enzyme';
import { Header } from './header';

describe('pages/phones/list/components/header specs', () => {
  it('should render as expected', () => {
    // Arrange

    // Act
    const component = shallow(<Header />);

    // Assert
    expect(component).toMatchSnapshot();
  });
});
