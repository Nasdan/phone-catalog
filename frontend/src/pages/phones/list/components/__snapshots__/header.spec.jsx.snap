// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`pages/phones/list/components/header specs should render as expected 1`] = `
<h1>
  Phone Catalog
</h1>
`;
